src/passes/SafeHeap.cpp

```diff
@@ -102,13 +109,14 @@ struct AccessInstrumenter : public WalkerPass<PostWalker<AccessInstrumenter>> {
 
 struct SafeHeap : public Pass {
   PassOptions options;
+  Name funcToIgnore;
```

Comment:
This simultaneously looks like a general mechanism for ignoring functions (due to the name) and also like a very specific mechanism (because there is only one name). Would it make more sense to name it something like `shouldIgnoreGetSbrkPtr` and make it a bool to make it clear that it is a very specific mechanism that is only used for that one thing?

Reply:
A bool would be inconvenient actually, because we need to know the name. If we just pass a bool to a function-parallel pass like this, then each instance (one per function) would need to look up the function name, and we need to look it up because we need to find an export with that name - there may be no Name section, but the export is guaranteed.

Reply:
(I can add a comment explaining this more, if you think that's useful?)

Reply:
I see. Perhaps a name like `GetSbrkPtrFuncToIgnore` would be more descriptive then?

Reply:
Good point, yeah, a better name could help here. How about `getSbrkPtrName` (which matches `getSbrkPtr` which is the actual function), which I pushed now?

Reply:
Oh wait... `getSbrkPtr` was *already* the name. So no new variable or name was needed! Fixed.